code/game/objects/items/weapons/blades.dm

```diff
@@ -39,6 +39,17 @@
 	attack_speed = 12
 	w_class = WEIGHT_CLASS_BULKY
 
+//FC's sword. Based roghly off a german Dusack.
+
+/obj/item/weapon/claymore/mercsword/officersword
+	name = "\improper Officers sword"
+	desc = "This appears to be a rather old blade that has been well taken care of, it is probably a family heirloom. Oddly despite its probable non-combat purpose it is sharpened and not blunt."
+	icon_state = "officer_sword"
+	item_state = "officer_sword"
+	force = 70
```

Comment:
Give it machete numbers and nerf the machete.
Otherwise someone is gonna throw a synth this and we're gonna have to bwoink them.

Reply:
I'm tired of synth bullshittery. I'm going to fix that.